libs/react-spa/bff/src/lib/bff.hooks.ts

```diff
@@ -1,13 +1,15 @@
 import { AuthContext } from '@island.is/auth/react'
 import { createBroadcasterHook } from '@island.is/react-spa/shared'
 import { BffUser, User } from '@island.is/shared/types'
+import { isDefined } from '@island.is/shared/utils'
+import * as kennitala from 'kennitala'
 import { useContext, useMemo } from 'react'
 import { BffContext, BffContextType } from './BffContext'
 
 /**
  * Maps an object to a BffUser type.
  */
-export const mapToBffUser = (input: User): BffUser => {
+export const mapToBffUser = (input: User, iss?: string): BffUser => {
```

Comment:
FYI I am going to change this hook file to what it was with little changes since the application system is breaking in some places. Will update later today. Just be aware that this file is going to change

Reply:
Since we are moving application system in parallel then I will revert back to only support bff context 🙈